git-project

```diff
@@ -46,10 +50,61 @@ def usage(exitcode=0):
     finish(exitcode)
 
 
+def check_version(v1):
+    version_diff = 0
+
+    v1_arr = v1.split('.')
+    v2_arr = CURRENT_VERSION.split('.')
+
+    if v1_arr[0] > v2_arr[0]:
+        version_diff = 1
+    elif v1_arr[0] < v2_arr[0]:
+        version_diff = -1
+    elif v1_arr[1] > v2_arr[1]:
+        version_diff = 1
+    elif v1_arr[1] < v2_arr[1]:
+        version_diff = -1
+    elif v1_arr[2] > v2_arr[2]:
+        version_diff = 1
+    elif v1_arr[2] < v2_arr[2]:
+        version_diff = -1
+
+    if version_diff < 0:
+        print '\n WARNING: .gitproj uses older version of git-project. Consider upgrading.'
+    elif version_diff > 0:
+        print 'git-project install is out of date. .gitproj version: {}, git-project version: {}. Aborting'.format(using_version, CURRENT_VERSION)
+        finish(1)
+    else:
+        print 'Using .gitproj {}'.format(CURRENT_VERSION)
```

Comment:
Version checking seems great. Note that the `semantic_version` library allows you to do the above comparisons automatically (instantiate with a string, and it handles all of the operators). I think it might be better to use a lib in this case.